app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -17,7 +17,7 @@
 %>
 <div class="govuk-document-footer<%= direction_class %>">
   <h2 class="visuallyhidden">Document information</h2>
-  <div class="history-information">
+  <div class="history-information" id="history">
```

Comment:
Do we use this ID? It was there previously `<div class="change-notes" id="history">`, but I'm wondering if we can remove.